### PR TITLE
Cart management page

### PR DESCRIPTION
**Pull request by anamkaa**
Added cart management page
Screenshot (302)

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!!And the demo picture also shows it !!
Good work @anamkaa !!

File: page_layouts/cart-management.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    <link
+      rel="stylesheet"
+      href="https://fab-ui-lib.netlify.app/stylesheets/main.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="/stylesheets/style.css" />
+    <link rel="stylesheet" href="/stylesheets/page/cart-management.css" />
```
Comment (shuntzhere) [inline page_layouts/cart-management.html]: Try to import all different stylesheets to one main CSS file and then use it across your HTML pages, this way you don't have to repeat linking all the stylesheets.

File: stylesheets/page/cart-management.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+.gh-hr-card {
+  justify-content: space-between;
+  height: fit-content;
+  border-radius: 0.2rem;
+}
+
+.gh-cart-wrap {
+  display: flex;
+  justify-content: center;
+}
```
Comment (shuntzhere) [inline stylesheets/page/cart-management.css:12]: All the flex and its properties can be moved to utility classes of the component library.

**Review by shuntzhere — APPROVED** [review]
Very consistent class names. The files look good, although suggested some minor suggestions.

File: stylesheets/page/cart-management.css (hunk 2 above)
Comment (anamkaa) [reply stylesheets/page/cart-management.css:12]: Will implement that.

File: page_layouts/cart-management.html (hunk 1 above)
Comment (anamkaa) [reply page_layouts/cart-management.html]: okay

**Pull request merged**